### ISSUE TITLE
unequal margins on Overleaf

### ISSUE THREAD

**Issue by ll-O-ll** [jessexknight/ut-thesis#11]
Hey @jessexknight,

Basically, the margins aren't consistent from page to page. Is there a quick fix for this?

Thanks!

https://user-images.githubusercontent.com/34784027/145685881-01b63e53-a28b-490e-936c-8fac3ca36cc8.mp4

**Comment by jessexknight** [comment]
This behaviour _was_ intentional, for producing print+bound copies of a thesis, where the inner margins (i.e. alternating if printing two sided) should be larger for the binding. You may also notice some blank pages, since "important" pages are supposed to open on the right hand side. This is the default behaviour of the **book** class, from which ut-thesis inherits. More info in [the documentation](https://github.com/jessexknight/ut-thesis/blob/master/ut-thesis/ut-thesis.pdf).

However, this behaviour is probably unexpected, and unwanted by most users (and also SGS), so we updated the default behaviour this year (2021). Unfortunately, Overleaf currently uses the version as bundled in base TeX Live 2021, with the old behaviour. There is no way to update specific packages manually within Overleaf AFAIK.

Long story short, there are three options to fix:

1. Specify these options when loading the (old) class file on Overleaf: `\usepackage[oneside,equalmargins]{ut-thesis}`
2. Download the updated ut-thesis.cls file (with the updated defaults) from this repo and upload it to your Overleaf project, where it will supersede the TeX Live package,
3. Wait for TeX Live 2022 (~ April) which should include the updated version.

ps. would you mind editing the issue title to something like, "unequal margins on Overleaf", to help others find this issue? Thanks,
